docker-gc

```diff
@@ -241,11 +242,23 @@ xargs -n 1 $DOCKER inspect -f '{{.Image}}' 2>/dev/null |
 sort | uniq > images.used
 
 # List images to reap; images that existed last run and are not in use.
-$DOCKER images -q --no-trunc | sort | uniq > images.all
+echo -n "" > images.all
+$DOCKER images | while read line
+do
+    awk '{print $1};'
+done | sort | uniq | while read line
+do
+  $DOCKER images --no-trunc --format "{{.ID}} {{.CreatedAt}}" $line \
+    | sort --key 2 --reverse \
```

Comment:
I believe that the `--key` field indicates word to start sorting on per http://ccrma.stanford.edu/planetccrma/man/man1/sort.1.html,

> -k, --key=POS1[,POS2]
              start a key at POS1, end it at POS 2 (origin 1)

`man sort` on my local machine doesn't include that detail, however the sort order definitely is including the time component for images created on the same day, so I believe it is working as intended.